### DISCUSSION TITLE
What configs needed to create cluster with multiple buckets?

### DISCUSSION THREAD

**Discussion by kocmoc1** [seaweedfs/seaweedfs#4210] [Q&A]
I needed ti create cluster with no bucket and want to create buckets throwS 3 Api programmatically is it possible?

Seaweedfs installed in K8s, cluster creates bucket accordind s3 api url, like bucketName.my.domain.com.

**Comment by astrung** [comment]
First  you need enable a S3 server when create seaweed cluster. You can see one example to start s3 server in cluster by docker:
https://github.com/seaweedfs/seaweedfs/blob/master/docker/compose/local-cluster-compose.yml
After creating cluster, now you can start using S3 client to create bucket: 
https://github.com/seaweedfs/seaweedfs/wiki/AWS-CLI-with-SeaweedFS

**Reply by kocmoc1** [reply]
Thanks, it's clear. In docker its work fine, but in K8s through ingress url, couldn't create bucket.

**Reply by astrung** [reply]
First, let try to request s3 api through CLI first. If you can not get any respond, it looks like your S3 and filer servers don't working right. And you should upload detail error here.
